scripts/variantstore/docs/aou/cleanup/cleanup.md

```diff
@@ -0,0 +1,104 @@
+      previous dataset(s). In the AoU Delta timeframe, the Variants team additionally reached out to a contact at VUMC
+      to determine if various datasets from the Alpha / Beta eras were still in use or could possibly be deleted.
+* Prepare tables
+    * These tables are used for callset statistics only starting with Delta but were previously used for extract as well
+      in pre-Delta callsets. Per 2023-01-25 meeting with Lee it appears this VET prepare table and associated callset
+      statistics tables can be deleted once the callset statistics have been generated and handed off.
+* Terra workspace
+    * It seems that VAT workflows may generate large amounts of data under the submissions "folder". e.g. ~10 TiB of
+      data under this folder in the AoU 10K workspace (!). At the time of this writing the VAT process is not fully
+      defined so this item may especially benefit from updates.
+* Avro files (Delta onward)
+    * These are huge, several times larger than the corresponding Hail VDS. It's not clear that there's any point to
+      keeping these files around unless there was a bug in the Hail GVS import code that would require a patch and
+      re-import. Per 2023-01-25 meeting with Lee we have now deleted the Delta versions of these Avro files. Per the
+      preceding comments, going forward these files can be deleted once the Variants team feels reasonably confident
+      that they won't be needed for the current callset any longer.
+* Hail VariantDataset (VDS) (Delta onward)
+    * The Variants team creates a copy of the VDS and then delivers a copy to the AoU preprod datasets bucket. That copy
+      of the VDS seems to stay in the delivery location for at least a few days, but it's not clear if that copy gets
+      cleaned up after AoU later copies the VDS to a production bucket. The Variants team should not rely on this copy
+      of the VDS being available long-term. Per 2023-01-25 meeting with Lee, we have retained the oldest (with AI/AN +
+      controls) and most recent versions (without AI/AN or controls, corrected phasing and GT) of the Delta VDS. This
+      can serve as our team's guidance for how to handle multiple VDS versions going forward, though of course we can
+      always ask Lee for explicit guidance.
+
+## Internal sign-off protocol
+
+The Variants team currently has the following VDS internal sign-off protocol:
+
+* Generate a VDS for the candidate callset
+* Run validation on this VDS
+* Run `GvsPrepareRangesCallset` to generate a prepare table of VET data
+* Generate callset statistics for the candidate callset using the prepare VET created in the preceding step
+* Forward VDS and callset statistics to Lee for QA / approval
+
+
+## Storage versus regeneration costs
+
+### Prepare tables
+
+* Running
+  GvsPrepareRangesCallset: [$1,803.14](https://docs.google.com/spreadsheets/d/1fcmEVWvjsx4XFLT9ZUsruUznnlB94xKgDIIyCGu6ryQ/edit#gid=0)
+* Storing prepare data: $878.39 / month
```

Comment:
I think it would be more helpful to have the way to calculate these numbers (e.g. number of GB * $6) in addition to the specific costs from the last run (I picked that number out of thin air).